index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta name="description" content="Youtube is cloned as a project of Microverse Curriculum">
+        <meta charset="utf-8">
+        <title>Youtube Clone</title>
+        <link rel="stylesheet" href="CSS/reset.css">    
+        <link rel="stylesheet" href="CSS/styles.css"> 
+    
+    </head>
+
+    <body>
+    
+        <nav id="navbar">
+            
+            <div id="menu" >
+                <img src="images/menuButton.png" alt="Click to see menu">
+            </div>
+            <div id="logo" >
+                <img id="logoImage" src="https://www.hubogi.com/wp-content/uploads/2017/09/Hubogi-Youtube-Yeni-Logo-ve-Sayfa-D%C3%BCzeni-Degisikliklerini-Aclikadi.jpg" alt="Logo of Youtube" >
+            </div>
+
+            <div id="textBox" >
+                <input id="inputText" type="text" size="100"  placeholder="Search">
+            </div> 
+
+            <div id="search" >
+                <button id="searchButton">
+                    <img id="searchIcon" src="images/searchIcon.png" alt="Click to search" >
+                </button>
+                
+            </div>
+            <div id="upLoad">
+                <img src="images/upLoad.png" alt="upLoad Icon" >
+            </div>
+            <div id="signIn">
+                <img src="images/signIn.png" alt="signIn Icon">
+            </div>
+
+            <!--
+                Next part is can be actived if new version of Youtube is wanted to use
+            -->
+
+            <!-- 
+            <div id="camera">
+                <img src="images/cameraIcon.png" alt="Camera Icon" >
+            </div>
+            <div id="nineDotIcon">
+                <img src="images/nineDotIcon.png" alt="Apps Icon">
+            </div>
+            <div id="ring">
+                <img src="images/ringIcon.png" alt="Ring Icon">
+            </div>
+            <div id="profilIcon">
+                <img id="profilImage" src="images/profilIcon.png" alt="Avatar">
+            </div-->
+        </nav>
+        
+        <main>
+            <br>
+            <br>
```

Comment:
- [ ] Don't use `<br>` tags. Positioning should be handled by CSS. [Read more here.](https://webmasters.stackexchange.com/questions/79199/why-isnt-the-html-break-tag-acceptable-anymore-for-top-and-bottom-margins)